controller/Controller.php

```diff
@@ -58,6 +58,7 @@ function tutorsPage($param)
         if ($this->_db->checkAdmin($_SESSION['user_id'])['user_is_admin'] == 0) {
             $this->redirects();
         }
+        $_SESSION['isAdmin'] = 1;
```

Comment:
Should this be hardcoded?

Reply:
this is just a easy way to tell if a admin is logged in. This way only if a person gets access to the data table it will be set.